cmd/qspi/src/lib.rs

```diff
@@ -576,6 +578,9 @@ fn qspi(context: &mut ExecutionContext) -> Result<()> {
         ops.push(Op::Call(qspi_page_program.id));
         (Some(arr), qspi_page_program)
     } else if let Some(filename) = subargs.writefile {
+        if subargs.addr.is_some() {
+            bail!("--addr is not implemented when using --writefile");
```

Comment:
Nit, take it or leave it: couldn't this alternatively be done by putting `#[arg(..., conflicts_with = "addr")]` on the `writefile` argument?

Reply:
Sure, done!